src/NServiceBus.Core/Diagnostics/Metrics/MessagingMetricsFeature.cs

```diff
@@ -0,0 +1,32 @@
+namespace NServiceBus
+{
+    using System.Diagnostics.Metrics;
+    using Features;
+
+    /// <summary>
+    /// MessagingMetricsFeature captures messaging metrics
+    /// </summary>
+    class MessagingMetricsFeature : Feature
+    {
+        internal static readonly Meter NServiceBusMeter = new Meter(
+            NServiceBusDiagnosticsInfo.InstrumentationName,
+            NServiceBusDiagnosticsInfo.InstrumentationVersion);
```

Comment:
I think the suggestion was to keep these versions separate (Core vs ActivitySource vs Metrics). Currently, the versions are shared between Metrics and Activities.